alf/trainers/off_policy_trainer.py

```diff
@@ -85,8 +85,11 @@ def __init__(self, config):
         self._driver_started = False
 
     def _init_driver(self):
-        for _ in range(1, self._config.num_envs):
-            self._create_environment()
+        for i in range(1, self._config.num_envs):
+            # [self._random_seed, self._random_seed + batch_size) has been used
+            # in policy_trainer.py
+            self._create_environment(
+                random_seed=self._random_seed + i * common._env.batch_size)
```

Comment:
Assuming batch_size == self._config.num_envs, maybe just use + i * self._config.num_envs, so code is more readable?

Reply:
num_envs is how many batched environments we have (only for async training). batch_size is the size of a batched environment